docs/features/feature-guides/properties/overview.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Overview
+---
+
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Structured Properties
```

Comment:
Do we usually say "DataHub X"? I usually try to drop that prefix sometiimes just seems like noise

Reply:
Same as before - we keep this for SEO, helps to have DataHub in H1